recipes/mysql-connector-odbc/build.sh

```diff
@@ -0,0 +1 @@
+cp ./lib/*.so ${PREFIX}/lib/.
```

Comment:
These libraries should be rebuilt, not copied.  Is the source code available for them?

Reply:
@scopatz. Yes, but each time I’ve tried building them it at best only worked on the machine it was built on

Reply:
That should not be the case for conda-forge since we build against a common & consistent set of ABIs